WordPress/Classes/ViewRelated/Post/PostNoticeNavigationCoordinator.swift

```diff
@@ -23,23 +23,13 @@ class PostNoticeNavigationCoordinator {
                 return
         }
 
-        if FeatureFlag.postPreview.enabled {
-            let controller = PostPreviewViewController(post: page)
```

Comment:
@bjtitus - I think this was a bug in the original code, since we were showing `PostPreviewViewController` when `FeatureFlag.postPreview.enabled`.  So I just left the `else` condition instead.

Can you double-check that the change is fine?